report_thesis/src/sections/results/initial_results.tex

```diff
@@ -0,0 +1,50 @@
+
+\gls{svr} emerges as the best-performing model for $\ce{FeO_T}$ with the lowest \gls{rmsecv} of 2.847, while ridge regression with an \gls{rmsecv} of 3.385 and \gls{ngboost} with an \gls{rmsecv} of 3.561 also demonstrate competent performance.
+The standard deviation values indicate that while \gls{svr} has lower variability, the other models also maintain reasonable consistency.
+
+For $\ce{MgO}$, \gls{svr} exhibits the best performance with the lowest \gls{rmsecv} of 1.426, and ridge regression with an \gls{rmsecv} of 1.509 and \gls{ngboost} with an \gls{rmsecv} of 1.578 are also effective.
+The standard deviation metrics suggest that these models are stable, with \gls{svr} showing the least variation.
+
+\gls{gbr} shows the lowest \gls{rmsecv} of 1.468 for $\ce{CaO}$, indicating strong performance, with ridge regression having an \gls{rmsecv} of 1.513 and \gls{etr} having an \gls{rmsecv} of 1.503 also performing well.
+Standard deviation values also highlight \gls{gbr} and \gls{etr} as having consistent performance.
+
+\gls{etr} demonstrates the best performance for $\ce{Na2O}$ with the lowest \gls{rmsecv} of 1.028, while ridge regression with an \gls{rmsecv} of 1.221 and \gls{svr} with an \gls{rmsecv} of 1.096 also perform reasonably well.
+The standard deviation of the cross-validation for \gls{etr} is particularly low, underscoring its robustness.
+
+For $\ce{K2O}$, \gls{etr} again shows strong performance with the lowest \gls{rmsecv} of 0.681, with \gls{svr} having an \gls{rmsecv} of 0.690 and ridge regression having an \gls{rmsecv} of 0.668 also performing well.
+The standard deviation analysis supports these findings, indicating reliable performance for these models.
+
+To summarize, \gls{svr} stands out as the best overall model based on its average performance, with a mean \gls{rmsecv} of 1.958. This is closely followed by \gls{ngboost} with a mean \gls{rmsecv} of 2.017 and ridge regression with a mean \gls{rmsecv} of 2.111.
+When examining the \gls{rmsep} values, \gls{svr} also demonstrates strong predictive accuracy on independent test sets, reinforcing its effectiveness.
+Additionally, the standard deviation of the cross-validation for \gls{svr} is generally low, indicating stable and consistent performance across different subsets of data.
+
+\gls{ngboost} and ridge regression also show robust performance, with relatively low standard deviations.
+Specifically, \gls{ngboost} exhibits consistent performance with low variability, making it a dependable model for various oxides.
+Ridge regression, while slightly higher in \gls{rmsecv}, still maintains good performance with low standard deviation values.
+
+Overall, \gls{svr}, ridge regression, and \gls{ngboost} are consistently among the top-performing models for multiple oxides, as evidenced by their low \gls{rmsecv}, \gls{rmsep}, and standard deviation values.
```

Comment:
I like this summary. However, I think what is missing is a conclusion on what models to proceed with per oxide.

For example since SVR is such a high performer its naturally on to the next round (could be used for all oxides), but do this for all the other models for each oxide.
Just as an aside - Whatever choice you make here is perfectly valid, so long as its based in their performance. Just make sure we have like 5-7 ish models that make it on the next round.

Reply:
and that they are diverse. So a good blend from each category would be preferable. (This is the argument we made in models selection - Stacking needs diverse models for each oxide)

Reply:
I can't agree with this summary. It doesn't match what I can see in the data nor the table.

(but I agree with SVR being good, it's top 2 on avg. performance measured by RMSECV!)

And why would 5-7 models 'make it to the next round' when we've obviously chosen 10? We ran the experiments for 10. See my comment on disqualifying only the worst 3, as their performance shows a steep increase in RMSECV compared to the other models.